src/tiles.js

```diff
@@ -0,0 +1,7 @@
+let tiles = {
```

Comment:
Try the "use strict" here. You get this error if not: `Block-scoped declarations (let, const, function, class)`